src/boot/efi/pe.c

```diff
@@ -163,8 +164,19 @@ static bool pe_section_name_equal(const char *a, const char *b) {
         return true;
 }
 
+static size_t pe_count_sections(
+                const PeSectionHeader section_table[],
+                size_t n_section_table,
+                const char *const section_name) {
+        size_t n = 0;
+        FOREACH_ARRAY(i, section_table, n_section_table) {
+                if (pe_section_name_equal((const char*) i->Name, section_name))
+                        n++;
+        }
+        return n;
+}
+
 static void pe_locate_sections(
-                const void *base,
```

Comment:
You added this argument in a previous commit, then removed it here. Please squash this into that previous commit to avoid the churn